distributed/deploy/tests/test_adaptive.py

```diff
@@ -203,7 +203,7 @@ async def test_adapt_quickly():
         assert len(adapt.log) == 1
 
         # Scale up when there is plenty of available work
-        futures = client.map(slowinc, range(1000), delay=0.100)
+        futures = client.map(slowinc, range(2, 1002), delay=0.100)
```

Comment:
what's the reasoning behind this change?

Reply:
`map` is rewriting the submitted function, i.e. it won't be registered as identical to the submitted one a couple lines above